Changes.md

```diff
@@ -4,7 +4,9 @@
 Improvements
 ------------
 
-- LightEditor : Values of inherited attributes are now displayed in the Light Editor. These are presented as dimmed "fallback" values.
+- LightEditor :
+  - Values of inherited attributes are now displayed in the Light Editor. These are presented as dimmed "fallback" values. Values are inherited from an ancestor of the inspected location or from attributes created in the scene globals.
```

Comment:
Might be worth mentioning the tooltip here.

Reply:
Good idea! I've mentioned the tooltip in 8c5812a.